pkg/kubelet/volume_host.go

```diff
@@ -188,6 +188,14 @@ func (kvh *kubeletVolumeHost) GetNodeLabels() (map[string]string, error) {
 	return node.Labels, nil
 }
 
+func (kvh *kubeletVolumeHost) GetNodeName() (string, error) {
+	node, err := kvh.kubelet.GetNode()
+	if err != nil {
+		return "", fmt.Errorf("error retrieving node: %v", err)
+	}
+	return node.GetName(), nil
```

Comment:
Why not just `return kvh.kubelet. nodeName`?

Reply:
I like that suggestion.  I can return types.NodeName and no need for err